data/south jobs.txt

```diff
@@ -0,0 +1,158 @@
+# Copyright (c) 2017 by Michael Zahniser
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+mission "Pirate Occupation [0]"
+	name `Pirates occupying <system>`
+	job
+	repeat
+	description `A small band of pirates have been occupying <system>, and the local government has asked for help. Defeat the pirates in the system and land on <planet> when done. Payment is <payment>.`
+	to offer
+		random < 20
+		"combat rating" > 100
+	source
+		attributes "south" "rim"
+	destination
+		distance 2 10
+		attributes "south"
+	npc evade
+		government "Pirate"
+		personality heroic staying target harvests plunders
+		system destination
+		fleet "Small Southern Pirates" 3
+	on complete
+		payment 100000
+		dialog `The government of <planet> thank you for defeating the pirate and pay you <payment>.`
```

Comment:
pirate -> pirates

Reply:
Also, it should be "thanks", since government is singular.